### PR TITLE
Add config as eslint, prettier, precommit

### PR DESCRIPTION
**Pull request by utopiaeh**
1. Added PULL_REQUEST_TEMPLATE
2. Added Eslint
3. Added Prettier
4. Added Precommit
5. Added SCSS compiler

P.S: Please in the future check necessary checkbox from pull template and in run plugin eslint and prettier.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,12 +1,13 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import * as serviceWorker from './serviceWorker';
+import React from 'react'
+import ReactDOM from 'react-dom'
 
-ReactDOM.render(<App />, document.getElementById('root'));
+import './index.css'
```
Comment (UncleFairy) [inline src/index.js]: css imports need to be in the end of import list

File: src/App.js [hunk 2]
```diff
@@ -1,15 +1,13 @@
...
-import logo from './logo.svg';
-import './App.css';
+import React from 'react'
+
+import logo from './logo.svg'
+import './App.css'
 
 function App() {
   return (
     <div className="App">
```
Comment (UncleFairy) [inline src/App.js]: delete all from div component

File: src/App.js [hunk 3]
```diff
@@ -1,15 +1,13 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from 'react'
+
+import logo from './logo.svg'
+import './App.css'
```
Comment (UncleFairy) [inline src/App.js]: add an empty line above './App.css' , it is a separate import group

File: src/index.js [hunk 4]
```diff
@@ -1,12 +1,13 @@
...
+import App from './App'
+import * as serviceWorker from './serviceWorker'
+
+ReactDOM.render(<App />, document.getElementById('root'))
 
 // If you want your app to work offline and load faster, you can change
 // unregister() to register() below. Note this comes with some pitfalls.
 // Learn more about service workers: https://bit.ly/CRA-PWA
```
Comment (UncleFairy) [inline src/index.js]: delete comments, please

**Review by CapceleaCristian — APPROVED** [review]

**Review by UncleFairy — APPROVED** [review]

**Review by andreiturcanu — APPROVED** [review]

**Pull request merged**